### PR TITLE
table-past-col.py pull request #1

### PR DESCRIPTION
**Pull request by jessicam1**

### CODE REVIEW

File: table-paste-col/table-paste-col.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Add a column containing a constant value to a table. Header assumed. 
+"""
+def get_input_file_object(filename):
+    if filename == "-":
+        return sys.stdin
+    return open(filename, "r")
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("-t", "--table", default='-', help="input csv file; STDIN if - (default: %(default)s")
+parser.add_argument("-c", "--colname", help="name for new column")
```
Comment (mnsmar) [inline table-paste-col/table-paste-col.py:15]: better use --col-name to maintain the same style as the perl script.

make this required `, required=True`

File: table-paste-col/table-paste-col.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+parser.add_argument("-v", "--colvalue", help="constant value for new column")
```
Comment (mnsmar) [inline table-paste-col/table-paste-col.py:16]: --col-val

Make this required

File: table-paste-col/table-paste-col.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+parser.add_argument("-s", "--separator", help="column separator character. use $ as prefix")
```
Comment (mnsmar) [inline table-paste-col/table-paste-col.py:17]: --sep

$ as prefix is only used for tabs. Other separators don't need this eg ",". I would remove this note from here as it is a Bash thing, not a concern of this script. 

use a default by adding `, default="\t"`

File: table-paste-col/table-paste-col.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+args = parser.parse_args()
+
+table = get_input_file_object(args.table)
+    
+def add_col(sep, colname, colvalue):
+    i = 0
+    for line in table:
+        if i==0:
+            line = line.rstrip('\n')
+            print(line + colname)
```
Comment (mnsmar) [inline table-paste-col/table-paste-col.py:27]: missing sep?

File: table-paste-col/table-paste-col.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+import sys
+import argparse 
+
+"""
```
Comment (mnsmar) [inline table-paste-col/table-paste-col.py:5]: Style: I think this needs to be above the imports

File: table-paste-col/table-paste-col.py (hunk 1 above)
Comment (jessicam1) [reply table-paste-col/table-paste-col.py:15]: If I use "col-name" and refer to it later with args.col-value I get a namespace error

Comment (mnsmar) [reply table-paste-col/table-paste-col.py:15]: you need args.col_value. argparse converts - to _

**Pull request merged**